### ISSUE TITLE
WHL files for Python 3.7

### ISSUE THREAD

**Issue by eLvErDe** [bcdev/jpy#168]
Hello,

Are you planning to release WHL files for Python 3.7 ?

Thanks in advance,

Adam.

**Comment by forman** [comment]
I'd like to, but currently have no project / spare time to do so.

Note that you can also build wheels on your own.

**Comment by devinrsmith** [comment]
@eLvErDe : my company (https://deephaven.io/) is maintaining a fork of jpy https://github.com/illumon-public/illumon-jpy. We have released 3.7 wheels, see https://pypi.org/project/deephaven-jpy/#files. Our release is a significant number of commits ahead of bcdev/jpy/master (and not *yet* reflected in our own master branch yet - we've got a bit of CI work on our internal servers and coordination that we need to do regarding releases, tags, and branches wrt our product version).

**Comment by devinrsmith** [comment]
@forman : do you have any interest in having Deephaven take an active role in maintaining jpy and producing artifacts?

**Comment by ShaheedHaque** [comment]
@devinrsmith I'm really glad you are actively maintaining a fork, and it would be great to see "official" new wheels for 3.6 etc.

**Comment by devinrsmith** [comment]
@ShaheedHaque : I wasn't originally responsible for our fork, so it might have been setup as an "unofficial" fork outside of the github fork mechanism. Definitely something I'll look into as we more formally establish our fork for public consumption. In the meantime - we do have newer 3.6 wheels :) - although, it doesn't look like the linux ones have been published to PyPi - I'll look into that.

**Comment by devinrsmith** [comment]
@ShaheedHaque : we have linux wheels available from our GitHub release page - https://github.com/illumon-public/illumon-jpy/releases. I've inquired about getting them published to PyPi as well - thanks!

**Comment by ShaheedHaque** [comment]
Cool. Let's see what @forman says in terms your kind offer.

**Comment by forman** [comment]
> @forman : do you have any interest in having Deephaven take an active role in maintaining jpy and producing artifacts?

@devinrsmith yes, thanks, we are definitely interested in contributions. However, we need to make sure that the evolution of jpy is still supporting our main use case, namely the beckend of the bidi Java-Python bridge [snappy](https://github.com/senbox-org/snap-engine/tree/master/snap-python) for our software project [SNAP](https://step.esa.int/main/toolboxes/snap/) which we develop for the European Space Agency.

I'm going to discuss your possible future role with the SNAP team, then I come back to you shortly.

**Comment by ShaheedHaque** [comment]
@forman any updates please?

**Comment by ShaheedHaque** [comment]
@forman any update on the proposal from @devinrsmith?

**Comment by JamesXNelson** [comment]
Deephaven continues to actively develop and maintain jpy.  If SNAP is willing to test our archives to ensure they are compatible, we can formalize our release process to include merging code upstream.  We are in the process of open sourcing part of our product, and are developing a new release process for our jpy wheels.  If there was a good time to consider handing off release management, that time would be now.

**Comment by devinrsmith** [comment]
@JamesXNelson - I believe we've started a discussion amongst our teams. Please check in w/ @chipkent

**Comment by pont-us** [comment]
> If there was a good time to consider handing off release management, that time would be now.

@forman suggests a meeting to discuss this and I have created a Doodle poll to find a suitable time in the near future. [Doodle poll link](https://doodle.com/poll/rgi9uenvskc84hya?utm_source=poll&utm_medium=link). Suggested slots should work for @forman, @marpet, and me -- hopefully one will also be OK for @JamesXNelson , @devinrsmith , @chipkent , and any other interested parties.

**Comment by forman** [comment]
> If SNAP is willing to test our archives to ensure they are compatible, we can formalize our release process to include merging code upstream. 

@JamesXNelson that sounds great. Lets dicuss our collaboration further. From the doodle, earliest possible date for @devinrsmith, @marpet, @pont-us any myself is Wednesday May 5, 14:00-15:00 CET. Let's go for it!
